centreon/tests/e2e/features/HostGroups/01-host-group-configuration/index.ts

```diff
@@ -0,0 +1,187 @@
+                name: services.serviceWarning.name,
+                template: services.serviceWarning.template
+            })
+            .addService({
+                activeCheckEnabled: false,
+                host: services.serviceOk.host,
+                maxCheckAttempts: 1,
+                name: services.serviceCritical.name,
+                template: services.serviceCritical.template
+            }).applyPollerConfiguration();
+
+        checkHostsAreMonitored([
+            { name: services.serviceOk.host },
+        ]);
+        checkServicesAreMonitored([
+            { name: services.serviceCritical.name },
+            { name: services.serviceOk.name }
+        ]);
+        cy.submitResults(resultsToSubmit);
+    });
+
+When('the user changes some properties of the configured host group "name, alias, comments"',
+    () => {
+        cy.navigateTo({
+            page: "Host Groups",
+            rootItemNumber: 3,
+            subMenu: "Hosts",
+        });
+        cy.wait('@getTimeZone');
+        cy.getIframeBody().contains(hostGroup).click({
+            force: true,
+        });
+
+        cy.waitUntil(
+            () => {
+                return cy
+                    .getByLabel({ label: 'Up status hosts', tag: 'a' })
+                    .invoke('text')
+                    .then((text) => {
+                        if (text !== '2') {
+                            cy.exportConfig();
+                        }
+
+                        return text === '2';
+                    });
+            },
+            { interval: 20000, timeout: 100000 }
+        );
+
+        cy.getIframeBody().find('input[name="hg_name"]').clear().type(hostGroup + '_new');
+        cy.getIframeBody().find('input[name="hg_alias"]').clear().type(hostGroup + '_new');
+        cy.getIframeBody().find('textarea[name="hg_comment"]').clear().type('Host group updated for test!');
+        cy.getIframeBody().find('input.btc.bt_success[name^="submit"]').eq(0).click();
+        cy.wait('@getTimeZone');
+    }
+);
+
+Then('the properties are updated',
+    () => {
+        cy.exportConfig();
+        cy.getIframeBody().contains(hostGroup + '_new').should('exist');
+    }
+);
+
+When('the user duplicates the configured host group',
+    () => {
+        cy.navigateTo({
+            page: "Host Groups",
+            rootItemNumber: 3,
+            subMenu: "Hosts",
+        });
+        cy.wait('@getTimeZone');
+        cy.getIframeBody().find('div.md-checkbox.md-checkbox-inline').eq(2).click();
+        cy.getIframeBody().find('select').eq(0)
+            .invoke('attr', 'onchange', "javascript: { setO(this.form.elements['o1'].value); this.form.submit(); }");
+        cy.getIframeBody().find('select').eq(0).select('Duplicate');
+        cy.wait('@getTimeZone');
+        cy.exportConfig();
+    }
+);
+
+Then('a new host group is created with identical properties',
+    () => {
+        cy.getIframeBody().contains(hostGroup + '_1').should('exist');
```

Comment:
where do you check that properties are identical ?

Reply:
the new duplicate host group has the same name as the first but a '_1' is added

Reply:
you are replacing the work `properties` by `name` :)
`properties` means name, alias, address, snmp community, linked templates...

Reply:
this test is based on what it has been done for behat test, they check just that they have the same name